test/suites/storage.sh

```diff
@@ -22,6 +22,35 @@ test_storage() {
   lxc storage volume show "$storage_pool" "$storage_volume" | sed 's/^description:.*/description: bar/' | lxc storage volume edit "$storage_pool" "$storage_volume"
   lxc storage volume show "$storage_pool" "$storage_volume" | grep -q 'description: bar'
 
+  # Test creating a storage pool from yaml
+  storage_pool_yaml="lxdtest-$(basename "${LXD_DIR}")-pool-yaml"
+  if [ "${lxd_backend}" = "btrfs" ] || [ "${lxd_backend}" = "zfs" ] || [ "${lxd_backend}" = "lvm" ]; then
```

Comment:
Can you expand the test to check for setting Description too please

Reply:
fixed